_redirects

```diff
@@ -39,16 +39,8 @@ layout: null
 /docs/v19.1/explore-the-admin-ui.html /docs/v19.1/admin-ui-overview.html 301
 /docs/v19.2/explore-the-admin-ui.html /docs/v19.2/admin-ui-overview.html 301
 /docs/v20.1/explore-the-admin-ui.html /docs/v20.1/admin-ui-overview.html 301
-/docs/v20.2/explore-the-admin-ui.html /docs/v20.2/admin-ui-overview.html 301
-/docs/v21.1/explore-the-admin-ui.html /docs/v21.1/admin-ui-overview.html 301
-/docs/v1.1/architecture/index.html /docs/v1.1/architecture/overview.html 301
```

Comment:
The existing "redirect_from: index.html" does not work anyway (possibly due to ambiguity with index.html at folder top-level).